lib/mockbuilder.py

```diff
@@ -195,12 +203,32 @@ def copy_results(self, package):
             self.timestamp, package.name)
         self._copy_rpms(package.build_cache_dir, package_build_results_dir)
 
+    def create_repository(self):
+        """
+        Create yum repository in build results directory.
+        """
+        result_dir = CONF.get('common').get('result_dir')
+        build_results_dir = os.path.join(
+            result_dir, 'packages', self.timestamp)
+        utils.run_command("createrepo %s" % build_results_dir)
+
+        repo_files_dir = os.path.join(result_dir, "repository_files")
```

Comment:
@mopsfelder I don't know if that's a good idea. With this structure, we wouldn't know if `result/latest` points to a `builds-packages` result or a `build-iso` result. The `repository_config` files that I'm creating here are also only useful in the system in which the build was executed. It would make no sense to move it around since the paths have to be hardcoded in yum config.